wikicharts/wikicharts.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #334.            plt.figtext(0.1, 0.025, "Graph Notes: Created by " + "Movement Insights" + " " + str(date.today()) + " using data from " + str(data_source), family=style_parameters['font'],fontsize=8, color= wmf_colors['black25'])
</pre><p>Why remove the configurability? Team name changes and reorgs do happen :)</p><p> </p><p>But, if you are going to remove it, you should remove it from parameters.py too, to avoid confusing others in the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>